conftest.py

```diff
@@ -32,39 +27,56 @@ def pytest_addoption(parser):
     parser.addoption(
         "--with-answer-testing",
         action="store_true",
-        default=False,
     )
     parser.addoption(
         "--answer-store",
         action="store_true",
-        default=False,
     )
     parser.addoption(
         "--answer-big-data",
         action="store_true",
-        default=False,
     )
     parser.addoption(
-        "--save-answer-arrays",
+        "--answer-raw-arrays",
+        action="store_true",
+    )
+    parser.addoption(
+        "--raw-answer-store",
+        action="store_true",
+    )
+    parser.addoption(
+        "--force-overwrite",
+        action="store_true",
+    )
+    parser.addoption(
+        "--no-hash",
         action="store_true",
     )
+    parser.addoption("--local-dir", default=None, help="Where answers are saved.")
+    # Tell pytest about the local-dir option in the ini files. This
+    # option is used for creating the answer directory on CI
+    parser.addini("local-dir", default="../answer-store", help="answer directory.")
```

Comment:
careful not to break windows paths here

Reply:
@neutrinoceros Just want to make sure I got the change right so that this conversation can be closed.

Reply:
Actually I'm not sure about your new solution. Let me close this thread and start a new one with the current implementation.